src/GitPrompt.ps1

```diff
@@ -101,6 +101,8 @@ $global:GitPromptSettings = [pscustomobject]@{
     DefaultPromptDebugSuffix                    = ' [DBG]$(''>'' * ($nestedPromptLevel + 1)) '
     DefaultPromptEnableTiming                   = $false
 
+    AbbreviateHomeDirectory                     = $true

```

Comment:
It's a bit long but still not the longest property name.  :-)  Folder is a bit shorter than Directory but I guess it doesn't matter that much.